py3status/modules/volume_status.py

```diff
@@ -149,6 +143,97 @@ def current_volume(self):
         # get info about channel mute status
         muted = self._get_muted(output)
 
+        return perc, muted
+
+    def volume_up(self, delta):
+        # volume up
+        cmd = 'amixer -q -D {} sset {} '.format(self.device, self.channel)
```

Comment:
For these, wouldn't it be simpler to do a line like this rather than passing through `shlex.split()`?
```cmd = ['amixer', '-q', '-D', self.device, self.channel, '{}%-'.format(delta)]```